webapp/src/components/schedule/WeekCalendar.js

```diff
@@ -74,6 +77,38 @@ class WeekCalendar extends PureComponent {
         calendar.download("Calendar");
     }
 
+    addToGCalendar(subsections) {
+
```

Comment:
instead of having this logic in WeekCalendar, it would be better to have it in the google calendar util class you made. Just pass in the subsections array there instead of doing the logic explicitly here.

This follows the object oriented paradigm of "tell not do", instead of outlining the steps explicitly here, we simply tell our google calendar class to add it and ideally the class should figure out what to do for itself.

In this manner, we can move the const variables inside GCalendar so WeekCalendar doesn't have to know about them and we can easily refactor as instead of modifying WeekCalendar, a pretty integral component, we just need to modify our util class.